docs/plugins/authors.asciidoc

```diff
@@ -3,12 +3,12 @@
 
 :plugin-properties-files: {elasticsearch-root}/build-tools/src/main/resources
 
-The Elasticsearch repository contains https://github.com/elastic/elasticsearch/tree/master/plugins/examples[examples of plugins]. Some of these include:
+The Elasticsearch repository contains https://github.com/elastic/elasticsearch/tree/main/plugins/examples[examples of plugins]. Some of these include:
```

Comment:
```suggestion
The Elasticsearch repository contains {es-repo}tree/main/plugins/examples[examples of plugins]. Some of these include:
```

Reply:
@abdonpijpelink thanks for the feedback changes applied